client/order/index.js

```diff
@@ -129,102 +134,176 @@ jQuery( function ( $ ) {
-			_n(
-				'(%s day left)',
-				'(%s days left)',
-				countdownDays,
+			// Disable the refund button.
+			refundButton.disabled = true;
+
+			const disputeDetailsLink = getDetailsURL( dispute.id, 'disputes' );
+			// TODO: finalize text copy after we come up with the final decision.
+			refundDisabledNotice = createInterpolateElement(
+				__(
+					'Refunds and order editing are disabled because this order was disputed. <a>View dispute</a>',
+					'woocommerce-payments'
+				),
+				{
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					a: <a href={ disputeDetailsLink } />,
+				}
+			);
+			let tooltipText = __(
+				'Refunds and order editing are disabled because this order was disputed.',
 				'woocommerce-payments'
-			),
-			countdownDays
-		);
-	}
+			);
 
-	const title = sprintf(
-		titleText,
-		amountFormatted,
-		reasons[ dispute.reason ].display,
-		dateI18n( 'M j, Y', dueBy.local().toISOString() )
-	);
+			if ( isAwaitingResponse( dispute.status ) ) {
+				refundDisabledNotice = __(
+					'Refunds and order editing are disabled during disputes.',
+					'woocommerce-payments'
+				);
+				tooltipText = refundDisabledNotice;
+			} else if ( isUnderReview( dispute.status ) ) {
+				refundDisabledNotice = createInterpolateElement(
+					__(
+						'This order has a payment dispute. Refunds and order editing are disabled during disputes. <a>View dispute</a>',
+						'woocommerce-payments'
+					),
+					{
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						a: <a href={ disputeDetailsLink } />,
+					}
+				);
+				tooltipText = __(
+					'Refunds and order editing are disabled during disputes.',
+					'woocommerce-payments'
+				);
+			}
 
-	// If the dispute is due within 72 hours, we want to highlight it as urgent/red.
-	if ( countdownDays < 3 ) {
-		urgency = 'error';
+			// Change refund tooltip's text copy.
+			for ( const maybeSibling of refundButton.parentNode.children ) {
+				if ( maybeSibling === refundButton ) {
+					continue;
+				}
+				const tooltip = maybeSibling.querySelector(
+					'.woocommerce-help-tip'
+				);
+				if ( ! tooltip ) {
+					continue;
+				}
+				jQuery( tooltip ).tipTip( { content: tooltipText } );
+			}
+		}
 	}
 
-	if ( countdownDays < 1 ) {
-		buttonLabel = __( 'Respond today', 'woocommerce-payments' );
-		suffix = __( '(Last day today)', 'woocommerce-payments' );
-	}
-	return (
-		<BannerNotice
-			status={ urgency }
-			isDismissible={ false }
-			actions={ [
+	let showWarning = false;
+	let warningText = '';
+
+	if (
+		dispute.evidence_details?.due_by &&
+		// Only show the notice if the dispute is awaiting a response.
+		isAwaitingResponse( dispute.status )
+	) {
+		const now = moment();
+		const dueBy = moment.unix( dispute.evidence_details?.due_by );
+		const countdownDays = Math.floor( dueBy.diff( now, 'days', true ) );
+
+		// If the dispute is due in the past, we don't want to show the notice.
+		if ( ! now.isAfter( dueBy ) ) {
```

Comment:
Minor suggestion: use `isBefore` to improve readability.
```suggestion
		if ( now.isBefore( dueBy ) ) {
```